benchmarks/benchmark_runner.py

```diff
@@ -0,0 +1,127 @@
+import argparse
```

Comment:
Can we include comments at top of this file for what it is meant to do, example invocations, documentation for arguments?

Reply:
Added